worlds/hk/Options.py

```diff
@@ -133,11 +139,14 @@
     if option_name in hollow_knight_randomize_options:
         continue
     extra_data = {"__module__": __name__}
+    # some options, such as elevator pass, are appear in logic_options despite explicitly being
```

Comment:
I think this word was accidentally added:
```suggestion
    # some options, such as elevator pass, appear in logic_options despite explicitly being
```